state-chain/runtime/src/chainflip/solana_elections.rs

```diff
@@ -80,8 +124,14 @@ impl OnChangeHook<SolAddress, SolHash> for SolanaNonceTrackingHook {
 
 pub struct SolanaElectionHooks;
 
-impl Hooks<SolanaBlockHeightTracking, SolanaFeeTracking, SolanaIngressTracking, SolanaNonceTracking>
-	for SolanaElectionHooks
+impl
+	Hooks<
```

Comment:
Not right now, but I think we should rename this trait to ElectionHooks or something... to distinguish it from the pallet hooks.